### PR TITLE
#174 Gitlab integration

### LINKED ISSUE #174: Gitlab integration
**What?**  
Create a new integration page that explain how to use Gitlab with Conan support.
The approach is similar to use Travis CI.

### PR DESCRIPTION
**Pull request by uilianries**
- Add integration page to explain how to use Conan in Gitlab CI

Fixes #174

### CODE REVIEW

File: integrations/gitlab.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    image: lasote/conangcc63
+
+    build:
+      # Upgrade Conan version
+      before_script:
+        - sudo pip install --upgrade conan
+
+      script:
+        # Automatic detection of your arch, compiler, etc. Download dependencies
+        - conan install
```
Comment (memsharded) [inline integrations/gitlab.rst]: I think I much prefer the ``conan test_package`` approach, which is better for settings-build consistency. 
In fact, one of the big changes we did to the docs is to describe ``test_package`` as the main package creation method.

What do you think?

**Comment by memsharded** [comment]
Thanks @uilianries for this contribution, very interesting.

@lasote, looks good for me (aside a small comment to better use ``test_package`` if possible). Is it good for the recent changes in docs you are preparing?

**Comment by uilianries** [comment]
Hi! thanks about the comment.

I just followed the same style for Travis and Appveyor, but I really prefer to use `conan test_package`.  
I'll take the opportunity to change Travis and Appveyor.

**Comment by memsharded** [comment]
Yes, that is totally right, the thing that has to be corrected is the Travis docs, to account for it :)
Maybe ask first @lasote, he is preparing a big PR for the docs, just in case he already fixed it.

**Comment by uilianries** [comment]
I just uploaded a new version.  
I changed Travis and Gitlab to use `conan test_package`.  
I didn't change Appveyor, I have no idea if will work fine, just calling `conan test_package`  
I'll contact @lasote to check these points.

**Pull request merged**